resource-dispatcher/system_management/__init__.py

```diff
@@ -44,6 +44,10 @@ def execute_supervised(process):
         print("[System Management] Spawning new task engine")
         if "CONFIG_FILE" not in os.environ:
             sys.exit("Environment variable CONFIG_FILE must be defined or this application has nothing to do. Exiting.")
+        if "ENABLE_TRIGGERING" in os.environ and os.environ["ENABLE_TRIGGERING"].lower() == "false":
+            print("[System Management] Triggering is disabled by environment variable - halting here as the task engine has no work to do")
+            while True:
+                pass
```

Comment:
This loop means that the program will never really get out of this state until restarted, correct? If so, perhaps make it break out of the loop to check ever so often? For the use in OpenShift and what we do for LodeStar it's probably not an issue as we'll trigger a rollout of the container, but since this is in a generic repo we probably should consider those that may not trigger a rollout.

Reply:
Made it exit every five min since environment variables don't update while a process is running - should be picked up on a pod restart.